### PR TITLE
Updating database with photographers

### PR DESCRIPTION
**Pull request by nfrieden**
Includes csv with new photographer data and python script used to upload said data into SQL

### CODE REVIEW

File: backend/scripts/sqliteversion.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import sqlite3
```
Comment (anastasia) [inline backend/scripts/sqliteversion.py:1]: Nice. Can you rename this file to something like "updatephotographers_sqlite.py"? That way it'll be more clear what it is

**Review by anastasia — CHANGES_REQUESTED** [review]
Great! Just one small change requested

File: backend/scripts/sqliteversion.py (hunk 1 above)
Comment (nfrieden) [reply backend/scripts/sqliteversion.py]: Done!

**Review by anastasia — APPROVED** [review]
Great, thanks!

**Comment by anastasia** [comment]
🎉

**Pull request merged**